lib/backend/model/resource.go

```diff
@@ -98,6 +98,11 @@ func init() {
 		"networkpolicies",
 		reflect.TypeOf(apiv3.NetworkPolicy{}),
 	)
+	registerResourceInfo(
+		KindKubernetesNetworkPolicy,
+		"networkpolicies",
```

Comment:
Won't this cause the plural resource mapping to be overwritten to only reference K8s network policies?

Reply:
Yes!

Although, I think that map might be dead code now? Still a really good catch. I'll fix it.

Reply:
Yeah, I guess it's really only used for etcd mode. Maybe it's wrong to even have  an entry for K8s network policy in that case since K8s network policies are treated through the same keywords in etcd mode?

Reply:
Actually we might need to use it for its string representation: https://github.com/projectcalico/libcalico-go/blob/79bef43e1a47f8b63f4944550872e12a3bda7e75/lib/backend/model/resource.go#L163 so it should be fine. etcd I guess will just know everything as `networkpolicies` and won't have anything written for `kubenetworkpolicies`